test/apiScenario/__snapshots__/apiScenarioGeneratorTest.ts.snap

```diff
@@ -1262,6 +1262,7 @@ Object {
 
 exports[`ApiScenarioGenerator generate api scenario from swagger - storage 1`] = `
 Object {
+  "authentication": undefined,
```

Comment:
if undefined, we should not output this field

Reply:
It won't output in yaml, but it should define in object to fix the order in the output.